pkg/cliplugins/workspace/cmd/cmd.go

```diff
@@ -45,25 +45,25 @@ func NewCmdWorkspace(streams genericclioptions.IOStreams) (*cobra.Command, error
 
 	cmd := &cobra.Command{
 		Aliases:          []string{"ws", "workspaces"},
-		Use:              "workspace [--workspace-directory-server=] <current|use|list>",
+		Use:              "workspace [--scope=<personal|all>] <create|list|use|current|delete>",
 		Short:            "Manages KCP workspaces",
 		Example:          fmt.Sprintf(workspaceExample, "kubectl kcp"),
 		SilenceUsage:     true,
 		TraverseChildren: true,
 	}
 
 	opts.BindFlags(cmd)
-	kubeconfig, err := plugin.NewKubeConfig(opts)
-	if err != nil {
-		return nil, err
-	}
 
 	useCmd := &cobra.Command{
 		Use:          "use < workspace name | - >",
 		Short:        "Uses the given workspace as the current workspace. Using - means previous workspace",
 		Example:      "kcp workspace use my-worspace",
 		SilenceUsage: true,
 		RunE: func(c *cobra.Command, args []string) error {
+			kubeconfig, err := plugin.NewKubeConfig(opts)
```

Comment:
(No action required) What's the point of in-lining this if the argument it depends on is provided outside the function scope?

Reply:
`opts` fields have their values changed when the command line flags are parsed